client/signup/steps/domains/index.jsx

```diff
@@ -82,12 +82,45 @@ import { getExternalBackUrl, shouldUseMultipleDomainsInCart } from './utils';
 
 import './style.scss';
 
+const wpcomSubdomains = [
```

Comment:
```suggestion
// Referenced from WordAds_Ads_Txt
const wpcomSubdomains = [
```
Adding the context may help with future list syncing.